<!DOCTYPE html>
<html>
  <head>
    <meta name="pypi:repository-version" content="1.1">
    <title>Links for mojo-errors</title>
  </head>
  <body>
    <h1>Links for mojo-errors</h1>
    <a href="https://files.pythonhosted.org/packages/67/4a/613643d5d345dc4b561813657f18e8c46e0833a87171e823fa75eced48a4/mojo_errors-0.0.1-py3-none-any.whl#sha256=2fce119addbba21b2f59d0dff47c1267a1fc9dbc5e0788f89966a00cc35946e8" data-requires-python="&gt;=3.8,&lt;4.0" data-dist-info-metadata="sha256=17c2560eb9ca371d74d11c20ab2bffbb056ef2eda0e305507de62093603e5d78" data-core-metadata="sha256=17c2560eb9ca371d74d11c20ab2bffbb056ef2eda0e305507de62093603e5d78">mojo_errors-0.0.1-py3-none-any.whl</a><br />
    <a href="https://files.pythonhosted.org/packages/07/14/9253d4a9d41f28dc91b0a0830f05d251111c9e7f110f93008041d2b9abd9/mojo_errors-0.0.1.tar.gz#sha256=9f2bf0c11ef7f334e136f5259101e56f4ba68fb7ee6023e9b0bb253019a1bdf5" data-requires-python="&gt;=3.8,&lt;4.0" >mojo_errors-0.0.1.tar.gz</a><br />
    </body>
</html>
<!--SERIAL 19324742-->